apps/ui/src/components/ProposalVoteRankedChoice.vue

```diff
@@ -31,7 +31,7 @@ const selectedChoices = ref<number[]>(props.proposal.choices.map((_, i) => i + 1
             {{ proposal.choices[element - 1] }}
           </div>
           <div
-            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-white bg-skin-border px-2 text-center inline-block"
+            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-skin-text bg-skin-border px-2 text-center inline-block"
```

Comment:
It should use text-skin-link similar as the vote count
```suggestion
            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-skin-link bg-skin-border px-2 text-center inline-block"
```